.github/workflows/cs_check.yml

```diff
@@ -0,0 +1,31 @@
+name: "Coding Style Check"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'app'
+      - 'system'
+
+jobs:
+  build:
+    name: PHP ${{ matrix.php-versions }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
`fail-fast` won't apply since there is only one job to run, can remove.

Reply:
fixed.